django_geosource/tests/test_serializers.py

```diff
@@ -115,42 +115,42 @@ def test_data_representation_is_correct_with_one_column_coordinates(self):
             csv_source.settings["coordinate_reference_system"],
         )
 
-        def test_missing_lat_and_long_field_raise_errors(self):
-            csv = SimpleUploadedFile(name="test.csv", content=b"some content")
-            data = {
-                "file": [csv],
-                "_type": "CSVSource",
-                "name": "test",
-                "encoding": "UTF-8",
-                "coordinate_reference_system": "EPSG_4326",
-                "coordinates_field": "two_columns",
-                "field_separator": "semicolon",
-                "decimal_separator": "comma",
-                "char_delimiter": "doublequote",
-                "number_lines_to_ignore": 0,
-                "use_header": True,
-            }
-            serializer = CSVSourceSerializer(data=data)
-            with self.assertRaises(CSVSourceSerializer.ValidationError):
-                serializer.is_valid()
-                self.assertEqual(len(serializer.errors), 2)
```

Comment:
Not sure about those 3 lines

Reply:
I checked for ValidationError because that what is raise in serializer (https://github.com/Terralego/django-geosource/blob/master/django_geosource/serializers.py#L357). 

But I suppose it also works if tests are passing.